src/mixins/canvas_grouping.mixin.js

```diff
@@ -53,7 +53,7 @@
       if (activeSelection.contains(target)) {
         activeSelection.removeWithUpdate(target);
         this._hoveredTarget = target;
-        this._hoveredTargets = this.targets;
+        this._hoveredTargets = this.targets.concat();
```

Comment:
added a concat like you used (TIL that concat is faster than slice(), good to know)